examples/models/CNN_genome.py

```diff
@@ -0,0 +1,98 @@
+import math
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+
+def single_conv(in_channels, out_channels):
+    return nn.Sequential(
+        nn.Conv1d(in_channels, out_channels, 7, padding=3), 
+        nn.BatchNorm1d(out_channels), 
+        nn.ReLU(inplace=True)
+    )
+
+def double_conv(in_channels, out_channels):
+    return nn.Sequential(
+        nn.Conv1d(in_channels, out_channels, 7, padding=3), 
+        nn.BatchNorm1d(out_channels), 
+        nn.ReLU(inplace=True),
+        nn.Conv1d(out_channels, out_channels, 7, padding=3), 
+        nn.BatchNorm1d(out_channels), 
+        nn.ReLU(inplace=True)
+    )
+
+
+class UNet(nn.Module):
```

Comment:
Could we write the input/output expected for this model in the docstring please?